tests/wpunit/Tribe/Tickets/Event_RepositoryTest.php

```diff
@@ -0,0 +1,257 @@
+<?php
+
+namespace Tribe\Tickets;
+
+use Tribe\Events\Test\Factories\Event;
+use Tribe\Tickets\Test\Commerce\PayPal\Ticket_Maker as PayPal_Ticket_Maker;
+use Tribe\Tickets\Test\Commerce\RSVP\Ticket_Maker as RSVP_Ticket_Maker;
+
+class Event_RepositoryTest extends \Codeception\TestCase\WPTestCase {
+	use RSVP_Ticket_Maker;
+	use PayPal_Ticket_Maker;
+
+	/**
+	 * An array of events grouped by their cost.
+	 *
+	 * @var array
+	 */
+	protected $events = [];
+
+	public function setUp() {
+		// before
+		parent::setUp();
+
+		// your set up methods here
+		$this->factory()->event = new Event();
+	}
+
```

Comment:
Can you add at least one test that confirms one of the original Event Repo 'where' works as expected?

Reply:
I did in the latest commit.